patch-carthage-project.rb

```diff
@@ -8,9 +8,23 @@
 targets_to_remove.each do |target|
   target.remove_from_project
 end
+
+# 2) Prevent linking of nghttp2 library
+carthage_nghttp2_unlink_targets = [ "BoringSSL", "CgRPC", "SwiftGRPC" ]
+targets_to_unlink = project.targets.select { |target| carthage_nghttp2_unlink_targets.include?(target.name) }
+ldflags_to_remove = [ "-L/usr/local/Cellar/nghttp2/1.38.0/lib", "-lnghttp2" ]
```

Comment:
Would it be possible to make this more generic, removing any `LDFLAG` containing `nghttp2`?

Reply:
I can scan every target in the scheme if that is all right with you

Reply:
Sounds good, but I was more thinking of removing any flag that contains `nghttp2`, even if there is not an exact match.

Reply:
Done